config-connector/solutions/iam/kpt/folder-iam/README.md

```diff
@@ -0,0 +1,51 @@
+Folder-iam
+==================================================
+
+# NAME
+
+  folder-iam
+
+# SYNOPSIS
+
+  Config Connector compatible YAML files to grant a specific member a role (default to roles/resourcemanager.folderEditor) to an existing folder.
+
+# CONSUMPTION
+
+  Using [kpt](https://googlecontainertools.github.io/kpt/):
+
+  ```
+  kpt pkg get https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit.git/config-connector/solutions/iam/kpt/folder-iam folder-iam
+  ```
+
+# REQUIREMENTS
+
+  A working Config Connector cluster using a cnrm-system service account.
```

Comment:
Should mention that the service account needs the Security Admin role in the folder.

Reply:
Added. Folder Admin role is enough to apply the solution so I think we should use that.